server/models/users.py

```diff
@@ -104,9 +101,11 @@ def decode_auth_token(auth_token):
             payload = jwt.decode(auth_token, 'SECRET_KEY')
             return payload['sub']
         except jwt.ExpiredSignatureError:
-            return 'Signature expired. Please log in again.'
+            return -1
```

Comment:
Why removing the messages and turning them in to ints?

Reply:
I did this because I don't think it'll be helpful in sending errors out when we use routes that need to be logged in. For example the GetUser resource will throw a 500 if we don't properly handle the error. Basically what I'm saying is that I think it would be easier to check -1 then it would be to check if the string is 'Signature expired. Please log in again.'